clusterloader2/docs/create-new-test.md

```diff
@@ -0,0 +1,67 @@
+# Create a new performance test
+
+## Select and copy the baseline test job config
+
+A great majority of our tests lie at the [test-infra] repository. All the
+configs there are written as [ProwJobs]. There are two types of tests being used
+there:
+
+-   **periodic** which is a test that runs at given cron interval
+-   **presubmit** which may be used as a presubmit for GitHub pull requests
+
+And there are three scales at which those are run: 100, 500 and 5000 nodes. Once
+it's known what is to be achieved it's good to start with a test that is the
+closest to it. Here are a few good example starting points, all running
+[load-test][]:
+
+Presubmits:
+
+-   100 nodes: [pull-kubernetes-e2e-gce-100-performance]
+-   5000 nodes: [pull-kubernetes-e2e-gce-scale-performance-manual]
+
+Periodics:
+
+-   100 nodes: [ci-kubernetes-e2e-gci-gce-scalability]
+-   5000 nodes: [ci-kubernetes-e2e-gce-scale-performance]
+
+We propose to start with 100 node tests as they are less costly and much
```

Comment:
For ambigious ownership, https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml might be a good file to use.

Reply:
Added:
```
New scalability tests with ambiguous ownership should be added to
[sig-scalability-experimental-periodic-jobs.yaml].
```